src/Migrations/Utils/Functions.php

```diff
@@ -438,6 +442,98 @@ public static function create_embed_block(
         );
     }
 
+    /**
+     * Create a new Post Query block.
```

Comment:
"Post Query" is a bit confusing to me, did you mean "Query Loop"?

Reply:
Done!